sale_exception_financial_risk/data/sale_exception_data.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record model="exception.rule" id="sale_financial_risk_exceeds_sale_order_risk">
+            <field name="name">This sale order exceeds the sales orders risk</field>
+            <field
+                name="description"
+            >This sale order exceeds the sales orders risk</field>
+            <field name="active" eval="True" />
+            <field name="sequence" eval="40" />
+            <field name="model">sale.order</field>
+            <field name="code">
+risk_amount = obj.currency_id._convert(
+    obj.amount_total,
+    obj.company_id.currency_id,
+    obj.company_id,
+    obj.date_order
+    and obj.date_order.date()
+    or context_today(),
+    round=False,
+)
+if obj.partner_id.risk_sale_order_limit and (
+    (obj.partner_id.risk_sale_order + risk_amount) > obj.partner_id.risk_sale_order_limit
+): failed=True</field>
```

Comment:
Remember to not use this way, but direct expression assignation with `bool` if needed.